lib/ddtrace/contrib/sneakers/configuration/settings.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'ddtrace/contrib/configuration/settings'
+
+module Datadog
+  module Contrib
+    module Sneakers
+      module Configuration
+        # Default settings for the Shoryuken integration
+        class Settings < Datadog::Contrib::Configuration::Settings
```

Comment:
Great work on this! thank you for the contribution.

small nit: there were some small changes made to env var settings in a recent PR that will need to get updated in this PR

- can we make sure to add an `enabled` option in settings and the respective env var support in ext?. For enabled, should be very straightforward and can default to `true`. Any of the other contrib integrations should be a good guide for what to add in settings and ext

 - https://github.com/DataDog/dd-trace-rb/blob/f6eb36d06cf1ccce0c3b0c073bbececf45d79fe6/lib/ddtrace/contrib/faraday/configuration/settings.rb#L15-L18

- https://github.com/DataDog/dd-trace-rb/blob/f6eb36d06cf1ccce0c3b0c073bbececf45d79fe6/lib/ddtrace/contrib/faraday/ext.rb#L7

Reply:
Thanks for the tipp.
I added the option. Yet I didn't quite understand how this option works.

Could you give me some pointers here?

Also, should I add it to the documentation? I saw at [one integration](https://github.com/DataDog/dd-trace-rb/blob/master/docs/GettingStarted.md#grape) has it yet the majority does not

Reply:
sure, so it will basically just not enable the integration if `enabled` is explicitly set to false: 
https://github.com/DataDog/dd-trace-rb/blob/f6eb36d06cf1ccce0c3b0c073bbececf45d79fe6/lib/ddtrace/contrib/extensions.rb#L59

Re: documentation, good question, oversight on my part on the last pr 😅 . I would add it to your documentation (ala the example u pointed to) and I'll put in a separate PR to update the rest of the docs. 

The env var approach is documented here: https://github.com/DataDog/dd-trace-rb/blob/master/docs/GettingStarted.md#environment-variables

Reply:
Thanks for the details explination :)

I added the option to the docs :)